src/docs/tutorials/04-edit-email-templates-with-flex-cli/index.md

```diff
@@ -188,19 +188,88 @@ text editor and make a simple change:
 
 When you've made the change, save the file.
 
+## Preview your changes
+
+You can test your changes and templates by previewing or sending a test
+email using your local template files. To preview the change above:
```

Comment:
How about breaking the preview rendering and test sending to separate chapters to better highlight that there are two different and complimentary ways to develop and test email templates. This occurred to me because the text has a bit weird flow where we first introduce both concepts on high level and then immediately follow with very specific "to preview the change above:"